bin/update-apis.js

```diff
@@ -145,77 +145,42 @@ program
   });
 
 program
-  .command('add <name> <reference-url>')
-  .option(
-    '-f, --force',
-    "Overwrite existing API entry with same name"
-  )
-  .description("Add API reference with <name> with <referenceUrl>")
-  .action(function(name, referenceUrl, options) {
-    // Check that we don't overwrite unless there is a force
-    if (apis[name] !== undefined && !options.force) {
-      console.log("API named: " + name + " already exists");
-      console.log("Use --force to overwrite it");
-      process.exit(1);
-    }
-
-    // Load reference from referenceUrl
-    console.log("Fetching reference from: " + referenceUrl);
-    var loaded_reference = request.get(referenceUrl).end().then(function(res) {
-      if (!res.ok) {
-        console.log("Failed to fetch reference from: " + referenceUrl)
-        console.log("Error: " + res.text);
-        process.exit(2);
-      }
-      return res.body;
-    });
-
-    // Save reference when loaded
-    loaded_reference.then(function(reference) {
-      apis[name] = {
-        referenceUrl: referenceUrl,
-        reference:    reference
-      };
-      updateDocs();
-      saveApis();
-      browserifyModule();
-      console.log("Add reference: " + name);
-    }).catch(function(err) {
-      console.log("Error adding reference: " + err);
-    });
-  });
-
-program
   .command('update')
   .description("Update all API references")
   .action(function() {
-    // Update remaining references
-    Promise.all(Object.keys(apis).map(function(name) {
-      var api = apis[name];
-      // Load reference from referenceUrl
-      console.log("Fetching reference from: " + api.referenceUrl);
+    // Fetch the Reference Manifest
+    var manifestUrl = 'http://references.taskcluster.net/manifest.json';
+    console.log('Fetching manifest reference from %s', manifestUrl);
+    var p = request.get(manifestUrl).end();
+
+    p = p.then(function(res) {
+      var manifest = res.body;
+      return manifest;
+    });
 
-      return request.get(api.referenceUrl).end().
-        then(function(res) {
-          if (!res.ok) {
-            console.log("Failed to fetch reference from: " + api.referenceUrl);
-            console.log("Error: " + res.text);
-            process.exit(2);
-          }
-          if (_.isEqual(api.reference, res.body)) {
-            console.log("No changes from: " + api.referenceUrl);
-          } else {
-            console.log("Received changes from: " + api.referenceUrl);
-          }
-          api.reference = res.body;
+    p = p.then(function(manifest) {
+      return Promise.all(Object.keys(manifest).map(function(name) {
+        console.log('Fetching %s reference', name); 
+        return request.get(manifest[name]).end().then(function(res) {
+          console.log('Fetched ' + name);
+          apis[name] = {
```

Comment:
When using manifest like this we should reset `apis = {}` when doing an update. Otherwise we will never remove entries...